app/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/ContactSAController.scala

```diff
@@ -23,23 +23,19 @@ import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.config.AppConfig
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.auth.AuthAction
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.forms.EnrolCurrentUserIdForm
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.logging.EventLoggerService
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.logging.LoggingEvent._
 import uk.gov.hmrc.taxenrolmentassignmentfrontend.models.{AccountDetails, MFADetails}
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.repository.SessionKeys.REDIRECT_URL
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.repository.TEASessionCache
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.services.{EACDService, UsersGroupSearchService}
-import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.{PTEnrolmentOnAnotherAccount, ReportSuspiciousID}
+
+import uk.gov.hmrc.taxenrolmentassignmentfrontend.views.html.{PTEnrolmentOnAnotherAccount, ContactSA}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class ReportSuspiciousIdController @Inject()(
-                                              authAction: AuthAction,
-                                              mcc: MessagesControllerComponents,
-                                              logger: EventLoggerService,
-                                              reportSuspiciousId: ReportSuspiciousID
+class ContactSAController @Inject()(
```

Comment:
I think we can have a clearer name for this page and controller. It's a reporting page that is relevant for if the user sees an account with a PT or SA Enrolment they don't recognise.  Something along the lines of `ReportAccount, ReportUserID` etc.